### ISSUE TITLE
Peer Review - Final Report

### ISSUE THREAD

**Issue by Wang-Siyuan** [anGie44/ORIE4741Project-HCMST#8]
This is a very interesting topic and the report is very well written. After glancing over your mid-term report, you have definitely experimented with more new approaches and improved your models.

Things I liked:
- You have shown a lot of effort that you put in cleaning and standardizing data. I really liked how you approached missing data. Instead of always predicting missing values, you also look at if that data entry is actually worth using or not.  That avoids the possibility of predicting most of the feature values in that data entry when that data entry is too sparse. 
- Throughout the report, you have tried many different ways to avoid overfitting. I think that is great. Especially given your data size, it was very important that you paid close attention to the in-sample and out-of-sample error comparison.
- I also liked how you use the results from a previous experiment to reason and select the next method to choose from, instead of randomly trying an array of methods.  The comparison between all the different loss function and regularizer was also helpful. 

Things that could improve:
- Although you included various attributes of your data, I didn't quite understand the exact input and output data for your model Are you just training on the participant from the first wave and response from the same participants in the 5th wave as your output data? It would be nice to include how many percents of users from the first wave still respond to the survey in the last wave.
- It would be nice if you could include whether your model's performance varies with the given data size.  Instead of always training on the 4002 data points, you could potentially start off training your model with fewer points, and compare the accuracy with your model when training with all data points available.

I think overall this was a quality project. The project group went through a series of data cleaning, model fitting and eventually achieved good prediction accuracy.  More importantly, we can make valid interpretation from the result and extract useful information. Great job.
